shell/common/rasterizer.cc

```diff
@@ -120,24 +120,16 @@ void Rasterizer::Draw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline) {
   }
   FML_DCHECK(task_runners_.GetRasterTaskRunner()->RunsTasksOnCurrentThread());
 
+  std::unique_ptr<LayerTree> layer_tree = layer_tree_holder->Get();
   RasterStatus raster_status = RasterStatus::kFailed;
-  Pipeline<flutter::LayerTree>::Consumer consumer =
-      [&](std::unique_ptr<LayerTree> layer_tree) {
-        raster_status = DoDraw(std::move(layer_tree));
-      };
+  if (layer_tree) {
```

Comment:
Nit. At this point a `?:` could be used to initialize raster_status in one line rather than set it and replace it on non-null.